### DISCUSSION TITLE
Request for "transfer" icon being used for transfers

### DISCUSSION THREAD

**Discussion by arpitbatra123** [twilio-labs/paste#3876] [Request: Icon] [answered]
### Description

We're building out  a transfers experience for Flex, and I was unable to find this icon in Paste. Can this please be added?
<img width="1019" alt="image" src="https://github.com/twilio-labs/paste/assets/21967563/781b50be-4f00-447f-889f-b7e49ae1f12b">

It is being used here - https://www.figma.com/file/lVI4BWk6IgBCtJw3EltHwd/Agent-Experience---Conversations?type=design&node-id=4585-26693&mode=design&t=RiMsglFzHdPOl38i-0

### Link to icon

https://www.streamlinehq.com/icons/core-line?search=transfer&icon=ico_UxHcwAJR969MTk7i
( I've eyeballed it, but not sure if it's the correct one )

### Icon type

- [X] UI icon
- [ ] Product icon

**Comment by serifluous** [comment]
Hey @arpitbatra123 ! We have this CallTransfer icon:

<img width="478" alt="Screenshot 2024-04-26 at 10 19 25 AM" src="https://github.com/twilio-labs/paste/assets/3821840/561fc840-dbf5-4b17-9878-617aa4031e00">

Is it used anywhere else in Flex? If not, we could update the glyph to the one you suggested (but use the same CallTransferIcon component).

**Comment by arpitbatra123** [comment]
@serifluous We're actually building the transfer experience for "non-call" types of tasks as well, so this icon won't be suitable for that use case. Maybe we need a new "TaskTransferIcon"?

**Reply by serifluous** [reply]
got it, can you submit a dependency request for the icon? https://airtable.com/appwvsQKyJHfnG5Sa/shrvMvL3BG6Zm0f9l We'll add it to our backlog—thanks!

**Comment by nkrantz** [comment, marked as answer]
Hey @arpitbatra123 - the icon is now available!
